serveur_calcul_python/tests/unit/test_rule_handling.py

```
@@ -0,0 +1,151 @@
+    pd.testing.assert_frame_equal(PR_result.units_table,PR_expect.units_table)
+
+def test_check_only_one_rule():
+    '''
+    # test_check_only_one_rule
+    This function checks that the validation function which verifies whether 
+    the ParkingRegulations object has only one rule functions correctly
+    '''
+    reg_unique = RDS.get_simple_rule_straight_conversion()
+    reg_multiple = RDS.generate_all_relevant_regs()
+
+    assert reg_unique.check_only_one_regulation()== True
```

Ruff (0.15.20)

[error] Avoid equality comparisons to `True`; use `reg_unique.check_only_one_regulation():` for truth checks

Replace with `reg_unique.check_only_one_regulation()`

(E712)